lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1348,10 +1357,26 @@ FIRStmtParser::parseExpWithLeadingKeyword(StringRef keyword,
 
   Value lhs;
   SmallVector<Operation *, 8> subOps;
-  if (lookupSymbolEntry(lhs, keyword, info.getFIRLoc()) ||
-      parseOptionalExpPostscript(lhs, subOps))
+  SymbolValueEntry symtabEntry;
+
+  if (lookupSymbolEntry(symtabEntry, keyword, info.getFIRLoc()))
     return ParseResult(failure());
 
+  // If we have a '.', we might have a symbol or an expanded port.  If we
+  // resolve to a symbol, use that, otherwise check for expanded bundles of
+  // other ops
+  if (resolveSymbolEntry(lhs, symtabEntry, info.getFIRLoc(), false)) {
```

Comment:
This is reachable with <= and other tokens, I think you should explicitly check for FIRToken::period here, something like:

`if (getToken.is(FIRToken::period)...`

Reply:
other tokens follow the same path as before.  I'll update the comment to be more clear.